runtime/oti/j9modifiers_api.h

```diff
@@ -118,4 +118,7 @@
 #define J9ROMMETHOD_IN_ITABLE(romMethod) \
 	J9_ARE_ALL_BITS_SET((romMethod)->modifiers, J9AccMethodVTable | J9AccPublic)
 
+#define J9ROMMETHOD_ALLOW_FINAL_FIELD_WRITES(romMethod, staticBit) \
+	((J9AccMethodAllowFinalFieldWrites | (staticBit)) == ((romMethod)->modifiers & (J9AccMethodAllowFinalFieldWrites | J9AccStatic)))
```

Comment:
Shouldn't this be
```
	((J9AccMethodAllowFinalFieldWrites | (staticBit)) == ((romMethod)->modifiers & (J9AccMethodAllowFinalFieldWrites | (staticBit))))
```
with `(staticBit)` on both sides? Otherwise the condition will never be true for uses where `staticBit == 0`.
It might be clearer to use `J9_ARE_ALL_BITS_SET`.